packages/exo/src/exo-makers.js

```diff
@@ -118,24 +140,24 @@ export const defineExoClassKit = (
     context.facets = facets;
     // Be careful not to freeze the state record
     freeze(context);
-    if (options) {
-      const { finish = undefined } = options;
-      if (finish) {
-        finish(context);
-      }
+    if (finish) {
+      // @ts-expect-error `facets` was added
+      finish(context);
     }
     return facets;
   };
+  // eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error -- different per package https://github.com/Agoric/agoric-sdk/issues/4620
```

Comment:
does the Endo repo suffer from this too?

Reply:
Just tested and it does not. I have no idea why they differ and I find it disturbing that they do. Nevertheless, in response I removed it, so I'll count this suggestion as "fixed". Resolving.